module/mock/dkgsdk_client_wrapper.go

```diff
@@ -0,0 +1,193 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
Should this be called just `sdk_client_wrapper.go`? It's shared between both QC and DKG clients right?